pkg/kubelet/cm/dra/manager.go

```diff
@@ -179,10 +183,13 @@ func (m *ManagerImpl) GetResources(pod *v1.Pod, container *v1.Container) (*Conta
 
 			klog.V(3).InfoS("add resource annotations", "claim", claimName, "annotations", claimInfo.annotations)
 			annotations = append(annotations, claimInfo.annotations...)
+			for _, cdiDevice := range claimInfo.cdiDevices {
+				cdiDevices = append(cdiDevices, kubecontainer.CDIDevice{Name: cdiDevice})
+			}
 		}
 	}
 
-	return &ContainerInfo{Annotations: annotations}, nil
+	return &ContainerInfo{Annotations: annotations, CDIDevices: cdiDevices}, nil
```

Comment:
I guess in the long run we can just do away with the `ContainerInfo` struct altogether and just pass `CDIDevices` back directly?

Reply:
That makes sense to me.